polyfill/lib/ecmascript.mjs

```diff
@@ -640,9 +640,11 @@ export const ES = ObjectAssign({}, ES2020, {
     return ES.GetOption(options, 'overflow', ['constrain', 'balance'], 'constrain');
   },
   ToTemporalOverflow: (options) => {
+    if (options === undefined) return 'constrain';
```

Comment:
Why make this change here, instead of adding `if (options === undefined) return fallback;` to `GetOption`?

Reply:
I didn't really look into that — GetOption is used in other places, for example in Intl.DateTimeFormat, and I assumed that this way would be less likely to cause unexpected behaviour changes elsewhere than changing GetOption would. I think it could probably be done that way if needed, though.

Reply:
Makes sense, was just wondering about the DRY benefit of doing the fallback in one place.